OpenRA.Mods.Common/Traits/Player/PlaceBuilding.cs

```diff
@@ -16,6 +16,8 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public class PlaceBuildingInit : IActorInit { }
```

Comment:
Can you please add a comment explaining that this is used by third party mods to distinguish the build source?